08-collab.md

```diff
@@ -97,7 +97,28 @@ Fast-forward
  1 file changed, 1 insertion(+)
  create mode 100644 pluto.txt
 ~~~
-
+>
+> ## Review of the Basic Collaborative Workflow using Git {.callout}
+>
+> A quick review of the basic collaborative workflow using is:
+> * track a change to the repository with `git add`
+> * stage the change using `git commit -m`
+> * upload the changes to GitHub with `git push origin master`
+> * downloand changes to our local copy of the repository using the `git push origin master`
```

Comment:
HI Ivan,
 Thanks for taking the time to review my changes.
 Yes that is a typo should be a “git pull origin master” but looking at it with fresh eyes and your feedback.
The first line should be
A quick review of the git commands with a basic collaborative workflow:

So should the first thing that some one should do is a “git pull” to update their local copy? Is there any 
harm with doing that?  
I have the git pull at the end so that the student can see that their repo is up-to-date.
Change the last line to 
“Verify that the local repo is up-to-date” using `git pull origin master`
something like that…

I’ll make some changes and resubmit.
Cheers,
Chris B.


> On 6 Jun 2016, at 5:26 AM, Ivan Gonzalez <notifications@github.com> wrote:
> 
> In 08-collab.md <https://github.com/swcarpentry/git-novice/pull/274#discussion_r65825416>:
> 
> > @@ -97,7 +97,28 @@ Fast-forward
> >   1 file changed, 1 insertion(+)
> >   create mode 100644 pluto.txt
> >  ~~~
> > -
> > +>
> > +> ## Review of the Basic Collaborative Workflow using Git {.callout}
> > +>
> > +> A quick review of the basic collaborative workflow using is:
> > +> * track a change to the repository with `git add`
> > +> * stage the change using `git commit -m`
> > +> * upload the changes to GitHub with `git push origin master`
> > +> * downloand changes to our local copy of the repository using the `git push origin master`
> There is a typo here git push origin master should be git pull origin master.
> 
> Also, do you think it would be a bit clear if you split the first three steps under an item, such as "Adding your changes to the common repo" and the fourth under another item, such as "Getting other people's changes from the common repo"? With your current wording it may seem that you should pull after pushing.
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub <https://github.com/swcarpentry/git-novice/pull/274/files/7bf8657c9920e33d6a8348dd3019ea2ec4ffa381#r65825416>, or mute the thread <https://github.com/notifications/unsubscribe/AIVsOaAX6DuveIFi05rRpZE08RaVsWOGks5qIz78gaJpZM4ImW9G>.
>